polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1306,6 +1306,13 @@ impl pallet_asset_rate::Config for Runtime {
 	type BenchmarkHelper = runtime_common::impls::benchmarks::AssetRateArguments;
 }
 
+// Notify `coretime` pallet when a lease swap occurs
+impl OnSwap for Runtime {
```

Comment:
Bold. Usually we create a helper struct for this. :grinning:

Reply:
Ups :) I didn't notice that. Fixed.